index.html

```diff
@@ -216,6 +216,34 @@ <h3 class="stack-title">Skills</h3>
         </li>
       </ul>
     </section>
+     <section class="contact">
+      <h3 class="form-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+      <form action="https://formspree.io/f/xvodqzjw" method="POST">
+        <ul class="form-list">
+          <li class="form-item">
+            <input class="form-input" type="text" name="name" id="name" placeholder="Full name" maxlength="30" required>
+          </li>
+          <li class="form-item">
+            <input class="form-input" type="email" name="email" id="email" placeholder="Email address" required>
+          </li>
+          <li class="form-item">
+            <textarea class="form-input textarea" name="message" id="message" cols="30" rows="10" placeholder="Enter text here" required></textarea>
```

Comment:
- [ ] I would kindly suggest you add the `maxlength="500"` to your text area as stated in the project requirement as this will give confidence in your ability to follow the requirement of any given project 👍 👍